meilisearch/index.py

```diff
@@ -1917,6 +1918,68 @@ def reset_search_cutoff_ms(self) -> TaskInfo:
 
         return TaskInfo(**task)
 
+    # PROXIMITY PRECISION SETTINGS
+
+    def get_proximity_precision(self) -> ProximityPrecision | None:
+        """Get the proximity_precision of the index.
+
+        Returns
+        -------
+        settings:
+            proximity_precision of the index.
+
+        Raises
+        ------
+        MeilisearchApiError
+            An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://www.meilisearch.com/docs/reference/errors/error_codes#meilisearch-errors
+        """
+        return self.http.get(self.__settings_url_for(self.config.paths.proximity_precision))
+
+    def update_proximity_precision(self, body: ProximityPrecision) -> TaskInfo:
```

Comment:
I like the way you currently have this, but just to be consistent with the other update methods let's allow `None` for the body.

```suggestion
    def update_proximity_precision(self, body: Union[ProximityPrecision, None]) -> TaskInfo:
```